bindings/qt/sdk.pri

```diff
@@ -323,7 +368,7 @@ win32 {
     LIBS += -lwinhttp -ladvapi32
     DEFINES += _CRT_SECURE_NO_WARNINGS
 }
-
+else:CONFIG += USE_CURL
```

Comment:
why this one? It does not seem to be used afterwards for !win32 anyway ....

Reply:
Yes I added this one because for vcpkg on non-win32, there was a build error (I forget if it was include or link), and the missing part was in one of the target-specific blocks.   Turning on USE_CURL means that the block on line 514 (inside vcpkg and protected by USE_CURL) adds include paths and libs for curl, openssl, and c-ares.   I'm not sure if this is the best balance, and taking into consideration mixing vcpkg and locally installed libraries for linux might mean we need some changes for that?  thanks